src/main/java/io/github/deweyjose/graphqlcodegen/Codegen.java

```diff
@@ -209,6 +227,29 @@ public void execute() {
                 return;
             }
 
+            if (typeMappingPropertiesFiles!=null && typeMappingPropertiesFiles.length > 0) {
```

Comment:
If we decide that the configuration of the plugin requires properties files to be scanned we can:


```
Set<Artifact> dependencies = project.getArtifacts();
            for (Artifact dependency : dependencies) {
                File artifactFile = dependency.getFile();
                if (artifactFile != null && artifactFile.isFile()) {
                    scanDependency(artifactFile);
                }
            }
```



```
 private void scanDependency(String artifactFile, String [] typeMappingPropertiesFile) {
        try (JarFile jarFile = new JarFile(artifactFile)) {
            for (String file : typeMappingPropertiesFile) {
                ZipEntry entry = jarFile.getEntry(file);
                if (entry!=null) {
                    try (InputStream inputStream = jarFile.getInputStream(entry)) {
                        // load the data into the typeMappingProperties
                    }
                }
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
```